examples/wow/zone_features__spark_v2.py

```diff
@@ -0,0 +1,60 @@
+import pandas as pd
+import pyspark.sql as ps
+from pyspark.sql import functions as sf
+
+from hamilton import htypes
+from hamilton.plugins.h_spark import with_columns
+
+
+def spark_session() -> ps.SparkSession:
+    return ps.SparkSession.builder.master("local[1]").getOrCreate()
+
+
+def world_of_warcraft(spark_session: ps.SparkSession) -> ps.DataFrame:
+    return spark_session.read.parquet("data/wow.parquet")
+
+
+def _flag_functions():
+    """Hidden from the DAG by using this wrapper function"""
+
+    def durotar_flag(zone: pd.Series) -> htypes.column[pd.Series, int]:
+        return (zone == " Durotar").astype(int)
+
+    def darkshore_flag(zone: pd.Series) -> htypes.column[pd.Series, int]:
+        return (zone == " Darkshore").astype(int)
+
+    return durotar_flag, darkshore_flag
```

Comment:
this is cute, but my preference here would be to put this into another module. That way it can be shared and reused more easily.

Reply:
I made some changes, is that what you had in mind?